apps/src/gamelab/levels.js

```diff
@@ -215,7 +215,7 @@ levels.custom = utils.extend(levels.sandbox, {
     "functionParams_n": null,
     "callMyFunction": null,
     "callMyFunction_n": null,
-    "return": null,
+    "return": null
```

Comment:
This object must be valid JSON (not just JavaScript) for the test to work.  Omitting trailing commas seems like a small price to pay to cover this.